ui/src/components/header.tsx

```diff
@@ -19,7 +19,7 @@
 
 import { zodResolver } from '@hookform/resolvers/zod';
 import { Link, useNavigate, useRouterState } from '@tanstack/react-router';
-import { CircleUser, Home, Menu } from 'lucide-react';
+import { Home, Menu } from 'lucide-react';
```

Comment:
Commit message: This is not a "gravatar" in the sense of https://gravatar.com/. I'd say "initials icon" or so instead.